shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumElectCommand.java

```diff
@@ -81,7 +81,8 @@ public int run(CommandLine cl) throws IOException {
         } catch (UnavailableException e) {
           return false;
         }
-        return leaderAddress.getHostName().equals(address.getHost());
+        return leaderAddress.getHostName().equals(address.getHost())
+                && leaderAddress.getPort() == address.getRpcPort();
```

Comment:
getRpcPort return `19998` leaderAddress.getPort() is `19200`, so it never be equal